lib/logic/admin/admin_cubit.dart

```diff
@@ -21,23 +23,20 @@ class AdminCubit extends Cubit<AdminState> {
     if (name.isEmpty || rank <= 0 || nSlotsKey.isEmpty) {
       emit(ParkingLotCreationFailure(
```

Comment:
It would be a very good practice if we are not putting text directly into code. All messages should be handled through localization. https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization